zerver/migrations/0198_preregistrationuser_invited_as.py

```diff
@@ -10,6 +10,15 @@
 def set_initial_value_for_invited_as(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
     PreregistrationUser = apps.get_model("zerver", "PreregistrationUser")
     for user in PreregistrationUser.objects.all():
+        """
+        According to PreregistrationUser model
+        INVITE_AS = dict(
+            MEMBER=1,
+            REALM_ADMIN=2,
+            GUEST_USER=3,
+        )
+        but here INVITE_AS['REALM_ADMIN'] = 1?
+        """
         if user.invited_as_admin:
             user.invited_as = 1     # PreregistrationUser.INVITE_AS['REALM_ADMIN']
```

Comment:
just wanted to confirm first that this comment is actually a typo?

Reply:
Good catch, this does seem to be a bug of some sort.

Reply:
I'll take care of figuring out what we need to do here.